2022-03-17 16:17:04,846 INFO    MainThread:124709 [wandb_setup.py:_flush():75] Loading settings from /share/etud/e2008984/.config/wandb/settings
2022-03-17 16:17:04,846 INFO    MainThread:124709 [wandb_setup.py:_flush():75] Loading settings from /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/settings
2022-03-17 16:17:04,846 INFO    MainThread:124709 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-17 16:17:04,846 INFO    MainThread:124709 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-17 16:17:04,847 INFO    MainThread:124709 [wandb_init.py:_log_setup():405] Logging user logs to /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/run-20220317_161704-2w1yosrb/logs/debug.log
2022-03-17 16:17:04,847 INFO    MainThread:124709 [wandb_init.py:_log_setup():406] Logging internal logs to /share/mastoc/projects/erasmus/dl_historical_images/master_thesis/utils/wandb/run-20220317_161704-2w1yosrb/logs/debug-internal.log
2022-03-17 16:17:04,847 INFO    MainThread:124709 [wandb_init.py:init():439] calling init triggers
2022-03-17 16:17:04,847 INFO    MainThread:124709 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-17 16:17:04,847 INFO    MainThread:124709 [wandb_init.py:init():458] re-initializing run, found existing run on stack: fup59csa
2022-03-17 16:17:04,850 INFO    MainThread:124709 [wandb_run.py:finish():1539] finishing run vibferreira/master_thesis/fup59csa
2022-03-17 16:17:04,850 INFO    MainThread:124709 [jupyter.py:save_history():429] not saving jupyter history
2022-03-17 16:17:04,850 INFO    MainThread:124709 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-17 16:17:04,851 INFO    MainThread:124709 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-17 16:17:04,851 INFO    MainThread:124709 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-17 16:17:04,852 INFO    MainThread:124709 [wandb_run.py:_restore():1674] restore
2022-03-17 16:17:05,304 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 915
  total_bytes: 915
}

2022-03-17 16:17:05,653 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 915
  total_bytes: 915
}

2022-03-17 16:17:06,062 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 915
  total_bytes: 4027
}

2022-03-17 16:17:06,164 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 915
  total_bytes: 4027
}

2022-03-17 16:17:06,266 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1007
  total_bytes: 4027
}

2022-03-17 16:17:06,367 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1470
  total_bytes: 4027
}

2022-03-17 16:17:06,469 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1470
  total_bytes: 4027
}

2022-03-17 16:17:06,571 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4027
  total_bytes: 4027
}

2022-03-17 16:17:06,673 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4027
  total_bytes: 4027
}

2022-03-17 16:17:06,775 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4027
  total_bytes: 4027
}

2022-03-17 16:17:06,876 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4027
  total_bytes: 4027
}

2022-03-17 16:17:06,978 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4027
  total_bytes: 4027
}

2022-03-17 16:17:07,080 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4027
  total_bytes: 4027
}

2022-03-17 16:17:07,591 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4027
  total_bytes: 4027
}

2022-03-17 16:17:08,034 INFO    MainThread:124709 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4027
  total_bytes: 4027
}
local_info {
}

2022-03-17 16:17:09,176 INFO    MainThread:124709 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-17 16:17:09,176 INFO    MainThread:124709 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-17 16:17:09,179 INFO    MainThread:124709 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-17 16:17:09,187 INFO    MainThread:124709 [wandb_init.py:init():492] starting backend
2022-03-17 16:17:09,187 INFO    MainThread:124709 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-17 16:17:09,194 INFO    MainThread:124709 [backend.py:ensure_launched():219] starting backend process...
2022-03-17 16:17:09,214 INFO    MainThread:124709 [backend.py:ensure_launched():224] started backend process with pid: 124953
2022-03-17 16:17:09,216 INFO    MainThread:124709 [wandb_init.py:init():501] backend started and connected
2022-03-17 16:17:09,232 INFO    MainThread:124709 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-17 16:17:09,232 INFO    MainThread:124709 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-17 16:17:09,232 INFO    MainThread:124709 [wandb_init.py:init():565] updated telemetry
2022-03-17 16:17:09,304 INFO    MainThread:124709 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-17 16:17:10,105 INFO    MainThread:124709 [wandb_run.py:_on_init():1759] communicating current version
2022-03-17 16:17:10,169 INFO    MainThread:124709 [wandb_run.py:_on_init():1763] got version response 
2022-03-17 16:17:10,169 INFO    MainThread:124709 [wandb_init.py:init():625] starting run threads in backend
2022-03-17 16:17:10,866 INFO    MainThread:124709 [wandb_run.py:_console_start():1733] atexit reg
2022-03-17 16:17:10,867 INFO    MainThread:124709 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-17 16:17:10,867 INFO    MainThread:124709 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-17 16:17:10,869 INFO    MainThread:124709 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-17 16:17:10,869 INFO    MainThread:124709 [wandb_init.py:init():664] run started, returning control to user process
2022-03-17 16:17:10,879 INFO    MainThread:124709 [wandb_watch.py:watch():47] Watching
